docs/src/cli/usage.md

```diff
@@ -16,7 +16,7 @@ FLAGS:
 
 OPTIONS:
         --color <color>                     [default: auto]  [possible values: Always, Auto, Never]
-        --config <config>                  A toml file to configure the behavior of selene [default: selene.toml]
+        --config <config>                  A toml file to configure the behavior of selene [default: .selene.toml]
```

Comment:
This isn't technically accurate because the default is both `selene.toml` and `.selene.toml`.